_studio/mfx_lib/scheduler/src/mfx_scheduler_core_ischeduler.cpp

```diff
@@ -683,5 +683,50 @@ mfxStatus mfxSchedulerCore::DoWork()
 } // mfxStatus mfxSchedulerCore::DoWork()
 
 
+mfxStatus mfxSchedulerCore::FindActiveSyncPoint(const void *pOwner, mfxSyncPoint *pSyncPoint)
+{
+    if (nullptr == pOwner || nullptr == pSyncPoint)
```

Comment:
Can we use macro like ```MFX_CHECK``` here?

Reply:
not used in scheduler. So I don't want to introduce it,